lms/session.py

```diff
@@ -14,17 +14,18 @@ def includeme(config):
 
     config.set_session_factory(
         SignedCookieSessionFactory(
+            # https://docs.pylonsproject.org/projects/pyramid/en/latest/api/session.html#pyramid.session.SignedCookieSessionFactory
             secret=config.registry.settings["session_cookie_secret"],
             secure=secure,
             # ``httponly=True`` is recommended by the Pyramid docs to protect
-            # the cookie from cross-site scripting vulnerabilities, see:
-            # https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/sessions.html
+            # the cookie from cross-site scripting vulnerabilities.
             httponly=True,
             # 30 days
             max_age=60 * 60 * 24 * 30,
+            # Disable autoexpiring sessions. Rely only on max_age
+            timeout=None,
```

Comment:
In a previous PR:

- Added a value for max_age

That could be tested checking the session survived a browser (chrome) restart.

- Removed the autoexpiring parameters.

That didn't have any effect because the values removed were actually the defatuls.


Now passing None has the desired effect:

```A number of seconds of inactivity before a session times out. If None then the cookie never expires. ```